templates/include/telemetry/mobileCommon.1.schema.json

```diff
@@ -11,7 +11,8 @@
   "type": "string"
 },
 "os": {
-  "type": "string"
+  "type": "string",
+  "enum": [ "Android", "iOS", "iPhone OS" ]
```

Comment:
we need to validate os if we are using it as a partition, this is a temporary solution until Bug 1455381 is implemented

cc @fbertsch this change effects focus-event and mobile-metrics. I was able to confirm that this is all `os` values found in `focus_event_longitudinal`, but I couldn't find a mobile-metrics dataset to check.

Reply:
nevermind, doing this differently